tests/test_nc_to_mmd.py

```diff
@@ -1812,6 +1812,36 @@ def test_check_attributes_not_empty__accepts_0(self):
         ncin.subswath = 0
         self.assertEqual(None, md.check_attributes_not_empty(ncin))
 
+    def test_check_feature_type__missing(self):
+        """ Test that the correct warning is issued if the CF
+        attribute featureType is missing.
+        """
+        md = Nc_to_mmd(self.fail_nc, check_only=True)
+        ncin = Dataset(md.netcdf_product, "w", diskless=True)
+        ncin.tull = "tull"
+        md.check_feature_type(ncin)
+        self.assertEqual(
+            md.missing_attributes["warnings"][0],
+            "CF attribute featureType is missing - one of the feature"
+            " types listed in Table 9.1 in https://cfconventions.org/"
+            "Data/cf-conventions/cf-conventions-1.10/cf-conventions."
+            "html#_features_and_feature_types should be used.")
+
+    def test_check_feature_type__wrong(self):
+        """ Test that the correct warning is issued if the CF
+        attribute featureType is wrong.
+        """
+        md = Nc_to_mmd(self.fail_nc, check_only=True)
+        ncin = Dataset(md.netcdf_product, "w", diskless=True)
+        ncin.featureType = "tull"
+        md.check_feature_type(ncin)
+        self.assertEqual(
+            md.missing_attributes["errors"][0],
```

Comment:
samme her.

Reply:
altså at `missing_attributes['errors']` og `missing_attributes['warnings']` gjøres om til dict type? Høres ut som mye arbeid for lav gevinst..? Tester er jo uansett hardkodet..

Reply:
Hvis det kommer en error eller warning FØR den vi tester for her, så vil `md.missing_attributes["errors"][0] == STRING` være en annen streng enn strengen vi tester for (STRING) . Mens `STRING in md.missing_attributes["errors"]`

Reply:
aha